src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ConnectionService.java

```diff
@@ -253,8 +260,16 @@ private void sendToRabbitOrExceptionLogs(final String gmcId, final String design
       log.info("Sending message to rabbit to remove designated body code");
       rabbitTemplate.convertAndSend(esExchange, routingKey, connectionMessage);
     } else {
-      exceptionService.createExceptionLog(gmcId, returnCode);
+      exceptionService.createExceptionLog(gmcId, exceptionMessage);
     }
+
+    //update elastic search index goes here
```

Comment:
I assume the call to the updateConnectionReceiver below satisfies "update elasticsearch index", if so can we remove the "goes here" from the comment? looks like a TODO :D